doc/user/content/sql/alter-sink.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "ALTER SINK"
+description: "`ALTER SINK` allows cutting a sink over to a new upstream relation without causing disruption to downstream consumers."
+menu:
+  main:
+    parent: 'commands'
+---
+
+`ALTER SINK` allows cutting a sink over to a new upstream relation without
+causing disruption to downstream consumers. This is useful in the context
+of [blue/green deployments](/manage/dbt/development-workflows/#bluegreen-deployments).
+
+## Syntax
+
+{{< diagram "alter-sink.svg" >}}
+
+## Details
+
+To alter the upstream relation a sink depends on and ensure continuity in data
```

Comment:
```suggestion
To alter a sink's upstream relation and ensure continuity in data
```

(I think "upstream" and "depends on" are redundant here.)

Reply:
(Also @morsapaes the `CREATE SINK` documentation says "underlying relation" where we say "upstream relation" here—we should bring them in sync! I think I like the new "upstream relation" term better.)

Reply:
Actually I feel like something about the wording is off here! It's not about altering _and_ ensuring continuity, it's about ensuring continuity _while_ altering, right?

```suggestion
To ensure continuity in data processing when altering a sink's upstream relation, Materialize must pick ...
```